requirements.txt

```diff
@@ -1,4 +1,6 @@
+#Python 2.7
```

Comment:
This should go in `setup.py`, not requirements.txt. The requirements themselves are not specific to any Python version.